frontend/src/app/main/ui/export.cljs

```diff
@@ -84,55 +88,57 @@
-                       [:svg {:view-box (dm/str x " " y " " width " " height)
-                              :width 24
-                              :height 20
-                              :version "1.1"
-                              :xmlns "http://www.w3.org/2000/svg"
-                              :xmlnsXlink "http://www.w3.org/1999/xlink"
+               (for [[index {:keys [shape suffix] :as export}] (d/enumerate @exports)]
+                 (let [{:keys [x y width height]} (:selrect shape)]
+                   [:div {:class (stl/css :selection-row)}
+                    [:button {:class (stl/css :selection-btn)
+                              :data-value index
+                              :on-click on-toggle-enabled}
+                     [:span {:class (stl/css :checkbox-wrapper)}
+                      (if (:enabled export)
+                        [:span {:class (stl/css-case :checkobox-tick true
+                                                     :global/checked true)} i/tick-refactor]
+                        [:span {:class (stl/css-case :checkobox-tick true
+                                                     :global/uncheked true)}])]
+
+                     [:div {:class (stl/css :image-wrapper)}
+                      (if (some? (:thumbnail shape))
+                        [:img {:src (:thumbnail shape)}]
+                        [:svg {:view-box (dm/str x " " y " " width " " height)
+                               :width 24
+                               :height 20
+                               :version "1.1"
+                               :xmlns "http://www.w3.org/2000/svg"
+                               :xmlnsXlink "http://www.w3.org/1999/xlink"
                                                    ;; Fix Chromium bug about color of html texts
                                                    ;; https://bugs.chromium.org/p/chromium/issues/detail?id=1244560#c5
-                              :style {:-webkit-print-color-adjust :exact}
-                              :fill "none"}
+                               :style {:-webkit-print-color-adjust :exact}
+                               :fill "none"}
 
-                        [:& shape-wrapper {:shape shape}]])]
+                         [:& shape-wrapper {:shape shape}]])]
 
-                    [:div {:class (stl/css :selection-name)} (cond-> (:name shape) suffix (str suffix))]
-                    (when (:scale export)
-                      [:div {:class (stl/css :selection-scale)}
-                       (dm/str (ust/format-precision (* width (:scale export)) 2) "x"
-                               (ust/format-precision (* height (:scale export)) 2) "px ")])
+                     [:div {:class (stl/css :selection-name)} (cond-> (:name shape) suffix (str suffix))]
+                     (when (:scale export)
+                       [:div {:class (stl/css :selection-scale)}
+                        (dm/str (ust/format-precision (* width (:scale export)) 2) "x"
```

Comment:
There is a typo change `"x"` for `"px"` but you can also use the `format-pixels` function